wsgi/odyssy/announcement/models.py

```diff
@@ -55,6 +55,6 @@ def get_model_type():
     key = models.AutoField(primary_key=True)
     start_date = models.DateTimeField(default=utils.get_today_start)
     end_date = models.DateTimeField(default=utils.get_today_end)
-    tags = TaggableManager()
+    tags = TaggableManager(blank=True)
```

Comment:
I think `null = True` will also be required, if you want to store `null` values in database.